### PR TITLE
Update HandsDescription.xml

### PR DESCRIPTION
**Pull request by PietroLiuzzo**
https://github.com/BetaMasaheft/Documentation/issues/865

### CODE REVIEW

**Comment by PietroLiuzzo** [comment]
please, apply changes on the branch directly.

File: pages/HandsDescription.xml [hunk 1]
```diff
@@ -26,17 +26,43 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+            <seg type="rubrication"/>
+            <date notBefore="1900" notAfter="1949">First half of the 20th century.</date>
+            <desc>
+                Careful, somewhat irregular; by a good scribe (name:
+                <persName role="scribe" ref="PRS11958WaldaM">Walda Māryām</persName>
+                ).
+            </desc>
         </handNote>
     </egXML>
+            <p>In practice, <tag>seg type="script"</tag> contains a categorization, as "square script" in the example, Gwelh or
```
Comment (DenisNosnitsin1970) [inline pages/HandsDescription.xml:44]: gwəlḥ

File: pages/HandsDescription.xml [hunk 2]
```diff
@@ -26,17 +26,43 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+            <desc>
+                Careful, somewhat irregular; by a good scribe (name:
+                <persName role="scribe" ref="PRS11958WaldaM">Walda Māryām</persName>
+                ).
+            </desc>
         </handNote>
     </egXML>
+            <p>In practice, <tag>seg type="script"</tag> contains a categorization, as "square script" in the example, Gwelh or 
+                similar and for <gi>desc</gi> contains any other or more general description of the script.</p>
+            <p>The categorization of "square script", "gwelh", "compressed script", etc. is based on S. Uhlig's works
```
Comment (DenisNosnitsin1970) [inline pages/HandsDescription.xml:46]: gwəlḥ

File: pages/HandsDescription.xml [hunk 3]
```diff
@@ -26,17 +26,43 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                and relates to very broad "palaeographic periods", not really to true, well-defined scripts. 
+                Thus a definition "broad script" is, according to this system, equivalent to mid-14 - mid-15th century.
```
Comment (DenisNosnitsin1970) [inline pages/HandsDescription.xml:48]: square script

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**